docs/pages/substitution-conditional-loading.md

```diff
@@ -0,0 +1,319 @@
+```
+
+Next, we need to import these modules in _index.js_ and randomly pick a specific
+translation to use its value to populate the `h1` tag in _index.html_.
+
+We could import each individual module and then write a simple algorithm to pick one
+of the translation values, in our example, that's an ok solution since we only have 3
+translation modules, but you can imagine how annoying it would be to do the same, with
+100 or even more modules.
+
+### The string substitution syntax
+
+Add the following to the _index.js_ file we created before:
+
+```js
+import i18n from "locale/#{lang}";
+
+document.getElementById("header").textContent = i18n.helloWorld;
+```
+@highlight 1
+
+This syntax might be already familiar and the reason is that it resembles the [ES2015 Template Literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) syntax,
+with the difference that the character before the curly braces is a `#` sign.
+
+The name between the curly braces (`lang` in our example) is a module name, called condition module, the `steal-conditional` extension will load it first and grab [its default export](https://developer.mozilla.org/en/docs/web/javascript/reference/statements/export#Using_the_default_export); if the value is not a `string` it will throw an error, otherwise this value is used to replace the module name along with the `#` sign and the curly braces.
+
+Let's say, `lang`'s default export value is the string `'foo'`, the conditional import in _index.js_ would be interpolated to:
+
+```js
+import i18n from "locale/foo";
+```
+
+> It is also possible for the conditional module to define multiple conditions via the `.` modifier:
+> ```js
+> import "locale/#{lang.helloWorld}";
+> ```
+> See `steal-conditional`'s [README](https://github.com/stealjs/steal-conditional/blob/master/README.md) for more details.
+
+
+### Create the condition module
+
+Create _lang.js_ with:
+
+```js
+const locales = ['en', 'es', 'hi'];
+
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+
+export default locales[getRandomInt(0, locales.length)];
+```
+
+The condition module creates a constant that holds the possible translation files to use,
+in our example there are 3 possible modules, then, the `getRandomInt` helper is defined,
+this function returns a random number between the `min` argument and `max`.
+
+Finally, `getRandomInt` is used to export a value of the `locales` array.
+
+Reload _index.html_ to see your changes.
+
+![screen shot 2017-01-05 at 16 14 46](https://cloud.githubusercontent.com/assets/724877/21694104/5a33f32e-d362-11e6-8468-cafd83b0a021.png)
+
+If you reload _index.html_ a couple more times, you should see the other translations as well:
+
+![screen shot 2017-01-05 at 16 14 39](https://cloud.githubusercontent.com/assets/724877/21694124/6e1d4b60-d362-11e6-9d67-32ac20cbc0f3.png)
+
+Pretty cool, don't you think?
```

Comment:
links to https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language